apex-squid/src/test/resources/metrics/comments.cls

```diff
@@ -0,0 +1,20 @@
+/*
+    Some license for this class 
+    * comment with star
+    this license uses "double quotes"
+ 
+ */
+/* another comment*/
+/**/
+//NOSONAR
```

Comment:
and also can add more scenarios..?

valid:
//a comment /* **/

/*
asd
Asd //aaaaaaaaaaaa
*/

this would be an issue
/*
asd*/
asd
*/